talent/onboarding/Non-EU_residents.md

```diff
@@ -88,4 +88,41 @@ If the candidate has a family, he needs to [ask for an appointment](https://w6.s
 
 With this document the candidate has to go to the closest clinic (Centro de Salud) to their house and ask for the Health card (Tarjeta sanitaria).
 
+## Spanish Residence Permit Renewal 
+
+We have to apply to renew the resident card within 60 days before the card expires. See below a list of documents we will need to renew the permit. 
+
+1. Company:
+   - Informe de Comercio
+
+2. Employee:
+   - Passport copy - all pages. We need to show them you didn’t leave the country more than 6 months per year.
+   - Resident card - copy
+   - Contract work - copy
+   - Last three payslips - copy
+   - Letter from HR department with a certification of the employee is working at the same position.
+   - Labor life certificate
+   - Authorization letting the lawyer presents the application
+   - Form to apply
+   - Model 038 - paid tax
+
+
+If the employee has a family, we will also need:
+
+3. Wife/Husband
+   - Passport copy - all pages. We need to show them your wife/husband didn’t leave the country more than 6 months per year.
+   - Resident card - copy
+   - Authorization letting lawyer presents the application
+   - Form to apply
+   - Model 038 - paid tax
+   - Marriage Affidavit 
+   
+4. Children
+   - Passport copy - all pages. We need to show them your children didn’t leave the country more than 6 months per year.
+   - Resident card - copy
+   - Authorization letting the lawyer presents the application
+   - Form to apply
+   - Model 038 - paid tax
+   - Declaration Living together
```

Comment:
Seems ok, I just don't know if it's called "Declaration of Living together" instead

Reply:
@tsolakoua check: https://europa.eu/youreurope/citizens/family/couple/registered-partners/index_en.htm